### ISSUE TITLE
Which chechpoint should I choose in style stransfer phase?

### ISSUE THREAD

**Issue by chenshen03** [Simon4Yan/Learning-via-Translation#20]
Thank you for your excellent work. 
During the style transfer, which epoch checkpoint should I choose? It seems that no difference can be seen between the generated image on different epochs.

**Comment by Simon4Yan** [comment]
Hi!

I think you could directly use the last checkpoint for testing. By the way, please try our PyTorch version of SPGAN [here ](https://github.com/Simon4Yan/eSPGAN).

**Comment by chenshen03** [comment]
Thank you for your reply!

I have tried the pytorch version of SPGAN, but it performs worse than the tensorflow version. 
Could you share some experimental results under pytorch version?

**Comment by chenshen03** [comment]
In addition, I have tried to adopt SPGAN in my own dataset. Experimental results (rank1-acc) under different epochs are as follows:

- Epoch(1)：0.9148；
- Epoch(2)：0.9228；
- Epoch(3)：0.9178；

It seems that training more epochs does not help improve accuracy.

**Comment by Simon4Yan** [comment]
Our extension [paper ](https://arxiv.org/pdf/1811.10551.pdf) use pytorch version for SPGAN, and achieves a little higher accuracy than using Tensorflow. 

How many training images you have? What is the resolution of your images?

**Comment by Simon4Yan** [comment]
@chenshen03 You could directly contact me by Wechat. You could write an e-mail to me and give me your wechat.
